### ISSUE TITLE
thermal energy, or not

### ISSUE THREAD

**Issue by JohnDenker** [ericmazur/PnPbook#36]
Chapter 20, page 547:  There is a string of problems here, centered around the concept of «thermal energy».  This is in some ways related to issue #35.

~~In a previous chapter, «thermal energy» was defined to be "energy that was transferred thermally".  In other words,~~

~~Δ(Eth) := (ΔE)th     [by definition of  Δ(Eth)]                   [1]~~

~~That is risky business, because the «thermal energy» that we see in parentheses on the LHS exists only in rather special situations, such as the infamous Slinktato™.  The bugs come home to roost on page 547.  If we plug equation [1](above) into equation 20.2, we find that (ΔE)th = W + Q.  However, Q is equal to (ΔE)th [by definition of Q], so equation 20.2 is evidently a wrong equation.  It says Q = W + Q.  It is grossly violated whenever there is nonzero W.~~
- On page 156 the «thermal energy» is defined as some subset of the «incoherent» energy and denoted E_th.  In most systems this is a physically impossible distinction, as discussed in item #35, but for now let's accept this as the definition of E_th.
- On page 523 the «energy transferred thermally» is denoted Q.
- The usual dictionary definition of «heat» in a physics context corresponds to energy transferred thermally, i.e. Q.  This is somewhat close to the notion of «heating» that seems to be favored in the PER literature.
- There is also a familiar, widely-used notion of «heat content» that more-or-less corresponds to «thermal energy content» i.e. E_th.   For example, NASA and NOAA scientists routinely talk about the «heat content» of the oceans.  In special cases the notion of «thermal energy content» makes sense, but in more general cases it doesn't.

So, there is tremendous potential for confusion here.  For example, consider a gas that expands against a thermally-insulating piston, doing work against another gas.  In accordance with equation 20.2, this is a transfer of thermal energy, but it is not a thermal transfer of energy.

The same bug appears in a slightly different guise in equations 20.3 and 20.4, which allege that E_th is a function of state.  It might be for a Slinktato™ ... but it absolutely cannot be for a gas (ideal or otherwise).

This is a classic misconception.  Every freshman who has toyed with thermodynamics has tried to develop a theory in which «heat» and «work» are thermodynamic potentials, i.e. functions of state.  All-too-often, they think they have succeeded ... but of course they never really succeed.  The physics says that neither TdS nor PdV is the gradient of any potential, except in trivial cases.

In the PER literature a holy war was waged against the word «heat», arguing the «heat» is not a noun.  As the saying goes, this was the wrong war, at the wrong place, at the wrong time, and with the wrong enemy.  The problem is not with the parts of speech;  the problem is with the _concept_.

```
*  I'll say it again:  Ideas are primary and fundamental.  Terminology is important only insofar as it helps us formulate and communicate the ideas.

* As a corollary:  If the ideas are wrong, superficial changes to the terminology aren't going to help.
```

In particular, using «thermal energy» as a euphemism for «heat» is not helpful, not even a little bit, because the idea is still wrong.  The question is not whether TdS is a noun;  the question is whether it is a scalar.  Hint:  It's not.

Constructive suggestion:  Oddly enough, the equations on page 547 are rather easy to fix.  The LHS of equations 20.3 and 20.4 is properly E, the plain old energy.  You can differentiate it to obtain dE = T dS - P dV  (subject to mild restrictions),  which more-or-less corresponds to equation 20.2.  (Repairing the wording that surrounds the equations will be a somewhat bigger job.)

Bedrock suggestion:  It would help to get students to understand that not every vector field is the gradient of some potential.  This will require a major effort.  This is central to understanding what you can (and cannot!) do with Kirchhoff's so-called «laws», and it is also central to understanding what you can (and cannot!) do with «heat» and «work».  Otherwise you're just playing whack-a-mole against an endless succession of misconceptions.

Some diagrams and other tricks that may help with this can be found at
   https://www.av8n.com/physics/non-grady.htm

**Comment by JohnDenker** [comment]
The book ~~systematically~~ mostly avoids multi-dimensional derivatives.  Instead it uses the integral formulation.  I can see good pedagogical reasons for that choice (even though when I'm doing calculations it's not my favorite way of doing things).  In any case, I'm not going to second-guess the decision.

Previously I said "not every vector field is the gradient of some potential" ... which expresses the idea using the derivative formulation.   To be a good team player, I should rephrase my suggestion using the integral formulation.   It goes like this:

Suppose we are integrating along some path.  Actually, suppose there are two paths, Γa and Γb.  They share the same starting-point and ending-point.  That is:

```
        Γa(1) = Γb(1) = x1, and
        Γa(2) = Γb(2) = x2
```

The interesting thing is, sometimes the value of the integral depends only on the endpoints x1 and x2.  We say the value is a _function_ of x1 and x2.  However, sometimes the value of the integral depends on every detail of the path.  We say the value is a **_functional**_ of Γ.

What's even more remarkable is that in the expression

```
       ΔE  ==  E(2) - E(1)  =  W + Q
```

E itself is a function of the thermodynamic state, and therefore ΔE is a function of two states, independent of how the path goes from 1 to 2 ... whereas neither W nor Q is a function of state(†).  W and Q are not state functions and cannot be written in terms of state functions, not even approximately, because they depend on every detail of the path Γ(†).

```
    (†) means:  except in trivial cases
```

For about a dozen reasons (pedagogical and otherwise) I recommend making this dependence explicit, every time W and Q appear.  That is, make a habit of writing them as functionals, explicitly:  W[Γ] and Q[Γ].  As a point of notation: Observe that a function such as E(1) is written using parentheses, whereas a functional such as W[Γ] is written square brackets.  For example:

```
       ΔE(1,2)  ==  E(2) - E(1)  =  W[Γ] + Q[Γ]
```

This is important.   Every freshman who looks at thermodynamics tries to invent W-like and Q-like   functions that are functions of state.  Sometimes they think they have succeeded,  but of course they never really succeed.

**Comment by JohnDenker** [comment]
The first paragraph in my previous comment ~~is~~ was overstated.  The book minimizes the use of partial derivatives, but does not eliminate them entirely.  For an example of partial derivatives, see the wave equation i.e. equation 16.51 on page 430.

For a more general discussion of partial derivative issues, see item #55.

**Comment by ericmazur** [comment]
You misread. It is "energy transferred thermally", NOT 'transfer of thermal energy' or some such thing. Energy can be transferred into our out of a system in several ways: through mechanical interactions (W, "work", i.e. in a coherent manner), through thermal interactions (Q, the definition of energy transferred thermally, see Section 20.1) or through radiation (not discussed in this chapter). 

If radiation enters a system, you don't increase its "radiation energy". If you transfer energy mechanically (work), you don't just change the system's mechanical energy. Likewise if you transfer energy thermally, you don't just change the system's thermal energy. Any of these transfers increase the system's internal energy ΔE, and what types of energy the transfer ends up changing depends on the constraints.

Maybe I should add a checkpoint to prevent others from falling in that trap.

**Comment by JohnDenker** [comment]
This is much worse than a misreading.

On page VII the book endorses the pedagogical principle of ideas before names.  Focusing on the terminological issue of "thermal transfer of energy" versus "transfer of thermal energy" misses the deeper conceptual point.

The point is that with rare exceptions, energy is fungible.  The energy of a system is the energy, period.  It is not generally possible to identify the "thermal energy content" any more than it is possible to identify the "radiant energy content".

Students arrive with deep-seated misconceptions about this, based on long experience with oversimplified special cases where there is a more-or-less well-defined "thermal energy content", e.g. adding or removing "heat energy" from a hot potato or a baby bottle.  Setting forth a legally-correct definition of Q is nice, but nowhere near sufficient.  Similarly a brief checkpoint is nowhere near sufficient to dispel this misconception.

#### Suggestion:

It is no easy task to put a fence around the concept of "thermal energy content" so as to confine it to the special cases where it actually makes sense.  This will require a boatload of examples and counterexamples, plus explanation of the underlying conceptual issues.

At the very least:  In any situation that is complicated enough to permit construction of a heat engine, it is not possible to define "thermal energy content".  To say the same thing another way, if it is possible to draw a thermodynamic cycle enclosing nonzero area, there is no such thing as "thermal energy content".

However, the notion of "heat content" will never go away entirely, because there are cases where it does make sense.    See e.g. the Slinktato™ discussion at http://www.av8n.com/physics/thermo/cramped.html   Sometimes "heat content" makes sense in the short term even if it doesn't make sense in the longer term. 

A more general discussion of "ideas before names" can be found in item #177.

**Comment by JohnDenker** [comment]
I seems a bit glib to say that I am misreading this passage.  The whole point of this item #36 is that it is not possible to read this passage correctly.  It is not possible for an expert who already knows what it is supposed to say, and even less possible for a student.  See also item #180 for another issue that makes this passage hard to follow.

It is simply incorrect to assert `It is "energy transferred thermally", NOT 'transfer of thermal energy'` as in the earlier comment.  Note the contrast:
- On page 156 we find E_th  defined as "thermal energy".  Furthermore, near the bottom of page 548 we find the words `change of thermal energy`.    Also,  "Thermal energy" appears in the index, as an entry unto itself, targeting a couple dozen occurrences in the book.   (See #181 for a catalog of index-related issues.)  So this concept is definitely being used, on pages 547-548 and elsewhere, despite what the earlier comment says.
- Meanwhile on page 532 we find Q defined as "energy transferred thermally".  So this concept is being used also.

The passage on pages 547-548 uses these concepts without explaining the relationship between them.

**Comment by JohnDenker** [comment]
Again, it seems a bit glib to say I am misreading this passage.   The whole point of this item #36 is that it is not possible to read this passage correctly.  Item #187 discusses another problem with this passage.

**Comment by JohnDenker** [comment]
Yet again, it seems a bit glib to say that I am misreading this passage.  A comment above asserts It is `"energy transferred thermally", NOT 'transfer of thermal energy'` but please look at the opening words of section 20.5 on page 547:  It talks about

> the transfer of mechanical energy

Perhaps the intent was to say "energy transferred mechanically" so as to be consistent with the idea of "energy transferred thermally" ... but that's not what it actually says.

There is no way a student could read this passage without getting wrong ideas.
